packages/components/src/DateTimePickers/MonthYearWrapper/MonthYearWrapper.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MonthPicker from '../MonthPicker';
+import YearPicker from '../YearPicker';
+import theme from './MonthYearWrapper.scss';
+
+function MonthYearWrapper(props) {
+
+	return (
+		<div className={theme.container}>
+			<div className={theme.months}>
```

Comment:
I think you can get rid of those extra divs.
Each component already output a div on top. You should either
* add a classname to identify them, so your wrapper can target them to style them
or
* accept classNames props, so you can pass your custom classes to them